api/model/plan_person.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const Base_model = require("./base_model");
+const Bookshelf = require('../service/database').Bookshelf;
+
+class PlanPerson extends Base_model {
```

Comment:
Use bookshelf relation model.hasmany for many to many relation between plan and person,